### PR TITLE
feat(context): Support unix context and daemonless

### LINKED ISSUE #1056: feat(CLI): daemonless envd build
## Description

We should investigate if we can support daemonless envd build.

Ref https://github.com/moby/buildkit#daemonless

---

**Message from the maintainers**:

Love this enhancement proposal? Give it a 👍. We prioritise the proposals with the most 👍.

### PR DESCRIPTION
**Pull request by gaocegege**
Fix #1056 

This PR adds two new image tensorchord/envd-from-scratch built by goreleaser, and tensorchord/envd built by the build.sh.

Users can use tensorchord:envd to run build daemonlessly:

```
docker run \                                                        
    -it \                                                                                       
    --rm \                                                                                      
    --security-opt seccomp=unconfined \                                                         
    --security-opt apparmor=unconfined \                                                        
    -e BUILDKITD_FLAGS=--oci-worker-no-process-sandbox \                                        
    --entrypoint /envd-daemonless.sh \                                                          
    -v /home/gaocegege/code/go/src/github.com/tensorchord/envd/examples/python-basic:/tmp/work t
ensorchord/envd:daemonless-rootless --debug build -p /tmp/work --output type=image,name=test   
```

### CODE REVIEW

**Review by VoVAllen — APPROVED** [review]
LGTM. What's the advantage of daemonless?

File: scripts/envd-daemonless.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
```
[the remark this answers is not in the document]
Comment (gaocegege) [reply scripts/envd-daemonless.sh:1]: Sure, I will add it.

**Comment by gaocegege** [comment]
> LGTM. What's the advantage of daemonless?

As @Xiaoaier-Z-L said, there is no need to launch a buildkitd deployment on Kubernetes. We can just create a job to build envd image.

**Comment by gaocegege** [comment]
Comments addressed, PTAL.

**Review by kemingy — APPROVED** [review]
/lgtm

**Pull request merged**